### DISCUSSION TITLE
search for the exact word under the cursor

### DISCUSSION THREAD

**Discussion by dlintw** [helix-editor/helix#9015] [Q&A] [answered]
In vim, '*' can quick jump to next exact same word.  In Helix, it can not work. the guide's method can only search word inside word.  For example, I just want to search 'on', it will move to words like 'con', 'ton', etc.

**Comment by gabydd** [comment]
you can rebind * to include make search word bounded to have it work that way

```toml
[keys.normal]
"*" = ["search_selection", "make_search_word_bounded"]
```

**Reply by dlintw** [reply]
In vim, it is case sensitive, how to make the search be case sensitive?

After your tip, I found I can use 2 or 3 keys to mapping the vim's '*' function.

**Reply by dlintw** [reply, marked as answer]
Simulate vim's * & # function, setup In config.toml

```
[editor.search]
smart-case = false

[keys.normal]
"*" = ["move_char_right", "move_prev_word_start", "move_next_word_end",
  "search_selection",  "make_search_word_bounded", "search_next"]
"#" = ["move_char_right", "move_prev_word_start", "move_next_word_end",
  "search_selection", "make_search_word_bounded", "search_prev"]
```
